share/spice/iplookup/iplookup.js

```diff
@@ -30,7 +37,7 @@
             templates: {
                 group: 'text',
                 options: {
-                    content: api_result.ddg.blacklists.length ? Spice.iplookup.content : null,
```

Comment:
@MrChrisW it should return an empty array if the property exists and it's an empty array.

The benefit of `getProperty()` is that you can pass along a more complex/deeply nested chain and if any of them along the way don't exist it will return `null` instead of throwing an error. If the final element doesn't exist, it will return `undefined` (as would a native check). If the property does exist, it will return the value of that property:

```js
var a = {
    b: [],
    c: ["hi", "there"]
};

DDG.getProperty(a, "b");
// returns []

DDG.getProperty(a, "b.test");
// returns undefined

DDG.getProperty(a, "c.nested.property");
// returns null (because c.nested is undef)
```

I'll make a note to update the examples and yes we should definitely document the return types for these functions!